packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -1076,6 +1070,8 @@ export class Table extends TableBase {
   private readonly tablePartitionKey: Attribute;
   private readonly tableSortKey?: Attribute;
 
+  private readonly secondaryIndexesSchema = new Map<string, SchemaOptions>();
```

Comment:
Above, we have a `private readonly secondaryIndexNames = new Set<string>();`.

Instead of creating a new field, can you instead re-use that one, just changing its name, and its type?

Reply:
Fixed it, You are right we don't need secondaryIndexNames.